src/main/kotlin/de/timo_reymann/mjml_support/tagprovider/BuiltInMjmlTagInformationProvider.kt

```diff
@@ -1164,7 +1226,11 @@ class BuiltInMjmlTagInformationProvider : MjmlTagInformationProvider() {
                         "space between cell and border"
                     ),
                     ATTRIBUTE_COLOR,
-                    ATTRIBUTE_CONTAINER_BACKGROUND_COLOR,
+                    MjmlAttributeInformation(
+                        "container-background-color",
+                        MjmlAttributeType.COLOR,
+                        "inner element background color"
+                    ),
```

Comment:
chore: Please refactor the container-background-color attributes into a constant like you did for `ATTRIBUTE_BACKGROUND_POSITION_*`